lib/matplotlib/backends/backend_ps.py

```diff
@@ -401,29 +401,11 @@ def _get_font_ttf(self, prop):
         font.set_size(size, 72.0)
         return font
 
-    def _rgba(self, im):
-        return im.as_rgba_str()
-
-    def _rgb(self, im):
-        h,w,s = im.as_rgba_str()
-
-        rgba = np.fromstring(s, np.uint8)
-        rgba.shape = (h, w, 4)
-        rgb = rgba[::-1,:,:3]
+    def _rgb(self, rgba):
+        h, w = rgba.shape[:2]
+        rgb = rgba[::-1, :, :3]
         return h, w, rgb.tostring()
```

Comment:
`_rgb` is still called.  It is used because the Postscript format doesn't support RGBA images -- matplotlib will still pass RGBA to `draw_image`, but we need to chop off the `A` to save it in the file.